hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -457,6 +457,9 @@ public class HMaster extends HBaseServerBase<MasterRpcServices> implements Maste
   public static final String WARMUP_BEFORE_MOVE = "hbase.master.warmup.before.move";
   private static final boolean DEFAULT_WARMUP_BEFORE_MOVE = true;
 
+  // Only for testing
+  private boolean isLocalRegionFlushed = false;
```

Comment:
Is it possible to not adding a flag which is only used to verify test result? I think you can add a CP method for the flushMasterRegion call, and use attach a CP to verify that whether the master is called.

Reply:
Does CP here mean coprocessor? Is there a similar implementation example? sir @Apache9